model/src/main/java/org/cloudfoundry/identity/uaa/oauth/jwk/JsonWebKey.java

```diff
@@ -71,6 +71,9 @@ public enum KeyOperation {
     public static String KTY = "kty";
     public static String ALG = "alg";
 
+    public static final String X5C = "x5c";
```

Comment:
The claims are described in https://www.rfc-editor.org/rfc/rfc7517 
They belong to the official parts of Json Web Key Set, what UAA offers in /token_key[s] endpoint
Examples can be found in https://www.rfc-editor.org/rfc/rfc7517#page-25

Should I extend to X509_Thumprint and X509_Certificate instead of x5t and x5c ?

Reply:
not needed finally

Reply:
@jaristiz @strehle and I confirm this is addressed.